fdbserver/fdbserver.actor.cpp

```diff
@@ -471,152 +474,192 @@ static void printHelpTeaser( const char *name ) {
 	fprintf(stderr, "Try `%s --help' for more information.\n", name);
 }
 
+static void printOptionUsage(std::string option, std::string description) {
+	static const std::string OPTION_INDENT("  ");
+	static const std::string DESCRIPTION_INDENT("                ");
+	static const int WIDTH = 80;
+
+	boost::algorithm::trim(option);
+	boost::algorithm::trim(description);
+
+	std::string result = OPTION_INDENT + option + "\n";
+
+	std::stringstream sstream(description);
+	if (sstream.eof()) {
+		printf(result.c_str());
+		return;
+	}
+
+	std::string currWord;
+	sstream >> currWord;
+
+	std::string currLine(DESCRIPTION_INDENT + ' ' + currWord);
+	int currLength = DESCRIPTION_INDENT.size();
```

Comment:
This length doesn't include the space and current word that's already been added. Is that intended?

Reply:
That is not intended. Good catch!